components/status-indicator/status-indicator.js

```diff
@@ -0,0 +1,99 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class StatusIndicator extends LitElement {
+
+	static get properties() {
+		return {
+			/**
+			 * State of status indicator to display. Valid values are: 'default', 'success', 'alert', and 'none'.
+			 */
+			state: {
+				type: String,
+				reflect: true
+			},
+			/**
+			 * Text for status indicator.

```

Comment:
So far we've been avoiding adding these comments in the code unless they are absolutely needed.  Usually they end up bloating the code unnecessarily.